nextflow.config

```diff
@@ -66,6 +66,10 @@ params {
     min_mapped_reads           = 5
     extra_star_align_args      = null
     extra_salmon_quant_args    = null
+    extra_kallisto_quant_args  = null
+    kallisto_default_fraglen   = 200
+    kallisto_default_fraglen_se  = 200
```

Comment:
Why do we need 2 parameters for this?

Reply:
Ah, I see. One is for the length and the other is for the standard deviation.

Reply:
Need to rename `--kallisto_default_fraglen_se` --> `kallisto_default_fraglen_sd`